worlds/gl/Locations.py

```diff
@@ -0,0 +1,1811 @@
+import typing
+
+from BaseClasses import Location
+
+
+class LocationData:
+    name: str = ""
+    id: int = 0
+    difficulty: int = 0
+
+    def __init__(self, name, id_, difficulty):
+        self.name = name
+        self.id = id_
+        self.difficulty = difficulty
+
+
+class GLLocation(Location):
+    game: str = "Gauntlet Legends"
+
+
+valley_of_fire: typing.List[LocationData] = [
```

Comment:
```suggestion
valley_of_fire: list[LocationData] = [
```
I know this is a result of the PR being open for so long, but you should still probably just update this and similar instances to use the newer typing